### DISCUSSION TITLE
[@types/nightwatch]  does not support method waitUntil

### DISCUSSION THREAD

**Discussion by yjf0824** [DefinitelyTyped/DefinitelyTyped#60925] [Issues with a @types package]
According to the document https://nightwatchjs.org/api/waitUntil.html. I can't find the api after install latest version of @types/nightwatch. Will method "waitUntil" support? Thanks.

[a comment by an automated account is not shown]

**Comment by vaibhavsingh97** [comment]
@yjf0824 I had created this PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60170/files adding all the missing APIs and Commands. I have one open review comment, which I needed to fix. Post that we will have a new version, which will resolve your issue.

**Reply by yjf0824** [reply]
Oh, thanks.
